cl/recap/tasks.py

```diff
@@ -1128,6 +1128,13 @@ def fetch_pacer_doc_by_rd(
     """
 
     rd = RECAPDocument.objects.get(pk=rd_pk)
+    # Check court connectivity, if fails retry the task, hopefully, it'll be
+    # retried in a different not blocked node
+    if not check_court_connectivity(rd.docket_entry.docket.court_id):
+        if self.request.retries == self.max_retries:
+            return
```

Comment:
I think I'd mark the status here as failed with a message of `f"Blocked by court: {court_id}" I think that'd help our users understand what's happening.

Reply:
Yeah, I've added to mark as failed with this message, also added `self.request.chain = None` before the return to avoid executing the following tasks.